endpoints/proxy/frontend.go

```diff
@@ -12,20 +18,21 @@ import (
 	"github.com/openziti/zrok/zrokdir"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	"net"
-	"net/http"
-	"net/http/httputil"
-	"net/url"
-	"time"
 )
 
 type FrontendConfig struct {
 	IdentityName string
 	ShrToken     string
 	Address      string
+	Tls          *TlsConfig
 	RequestsChan chan *endpoints.Request
 }
 
+type TlsConfig struct {
+	CertPath string
+	KeyPath  string
+}
```

Comment:
Any way we can have just one `TlsConfig` type without circular dependencies?

Reply:
moved up. I wish I remembered where I had tried putting it originally to get a circular dependency 🤔